src/walletSdk/Anchor/index.ts

```diff
@@ -401,6 +409,130 @@ export class Anchor {
     };
   }
 
+  /**
+  * Watch a transaction until it stops pending. Takes three callbacks:
+  * * onMessage - When the transaction comes back as pending_ or incomplete.
+  * * onSuccess - When the transaction comes back as completed / refunded / expired.
+  * * onError - When there's a runtime error, or the transaction comes back as
+  * no_market / too_small / too_large / error.
+  */
+  watchOneTransaction(params: {
```

Comment:
Let's move it to new watcher class after all. Considering that we now have 2 methods.
It feels like anchor class has way too much things going on right now

Reply:
Sounds good to me, I'll try creating a Watcher class